.travis.yml

```diff
@@ -2,9 +2,12 @@ sudo: false
 language: node_js
 node_js:
   - "node"
+dist: trusty
```

Comment:
Trusty is the default test environment now, so we don't need to specify it here.

I've needed `sudo: true` for Chrome headless to work in the past. That issue may be fixed now, though.